admin/routes/check-started.js

```diff
@@ -1,10 +1,16 @@
 const express = require('express')
 const router = express.Router()
 const { checkStarted } = require('../controllers/check-started')
+const winston = require('winston')
 
 router.route('/').all((req, res) => {
-  if (req.method !== 'POST') return res.sendStatus(405)
+  winston.info(`MTC-API: ${req.url} reached`)
+  if (req.method !== 'POST') {
+    winston.error(`MTC-API: ${req.url}: req.method is not POST - res.sendStatus(405) - ${JSON.stringify(req.method)} `)
+    return res.sendStatus(405)
+  }
   checkStarted(req, res)
+  winston.info(`MTC-API: ${req.url}: request.body: ${JSON.stringify(req.body)}`)
```

Comment:
please snip `req.body` at 200 chars to deal with DOS submissions

Reply:
Done, please check it.